sql_generators/glean_usage/common.py

```diff
@@ -159,6 +162,17 @@ def __init__(self):
         self.per_app_enabled = True
         self.cross_channel_template = "cross_channel.view.sql"
 
+    def override_skip(self):
+        """Files configured not to be overridden during generation."""
+        return [
+            file
+            for skip in ConfigLoader.get("glean_usage", "override", "skip", fallback=[])
+            for file in glob.glob(
+                skip,
+                recursive=True,
+            )
+        ]
```

Comment:
If you agree with renaming the config `skip_existing`...
```suggestion
    def skip_existing(self):
        """Existing files configured not to be overridden during generation."""
        return [
            file
            for skip_existing in ConfigLoader.get("glean_usage", "skip_existing", fallback=[])
            for file in glob.glob(skip_existing, recursive=True)
        ]
```